src/components/ConversationSettings/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -121,8 +120,6 @@ import {
 	getBridgeProcessState,
 } from '../../../services/matterbridgeService.js'
 
-Vue.directive('tooltip', Tooltip)
```

Comment:
Sadly, there is no ESLint rule to forbid it in Vue 2...

Reply:
I see what can be done